services/dcs.yaml

```diff
@@ -0,0 +1,28 @@
+name: dcs
+image: download-controller-service
```

Comment:
Currently, the name of the service is "download-controller-service". This name can be used to infer the docker image and the repository name. The current abbreviation is "dcs". Can be discussed whether the abbreviations should be used as names.

Reply:
name: download-...
abbr: dcs

Reply:
version

Reply:
Replaced image with name and name with shortname, added version.